projects/libaom/README.md

```diff
@@ -0,0 +1,52 @@
+# Submit a Patch to oss-fuzz repo
+
+## One-time Setup
+
+1.  Create github account if needed (with @google.com email address, preferably)
+    and log in.
+1.  To allow “git push” to work, you’ll have to add an SSH key:
+    https://help.github.com/articles/connecting-to-github-with-ssh/
+1.  Go to https://github.com/google/oss-fuzz and click on “Fork”.
+1.  Go to your own fork of the repo, which will be at
+    https://github.com/<git_username>/oss-fuzz
+1.  Click on “clone or download” and pick “Clone with SSH” method (I found that
+    easier to use for “git push”). Then copy that URL and run “git clone <URL>”
```

Comment:
Looks like these angle brackets need to be escaped (<URL> isn't visible in the preview).

Reply:
Done.
I'm not sure how to check the preview though.